WTWSMS/localisation/WtWSMS_religions.csv

```diff
@@ -84,6 +86,10 @@ donatism_FEATURES;
 DONATIST;Grand Bishop;Grand �v�que;;;;;;;;;;x
 d_donatism;Bishopric of Carthage;�v�ch� de Carthage;;;;;;;;;;x
 d_donatism_adj;Donatistic;Donatistique;;;;;;;;;;x
+bad_priest_donatism;Traditor;;;;;;;;;;x
+bad_priest_donatism_desc;This clergyman or spiritual guide is among the Traditores (literally "traitors"), the mortal enemies of all believing Donatists. Forsaken and an antithema to his faith, he is considered a non-person and can be arrested and disposed of at will. The Donatism religion orders all clergy deemed Traditores to immediately vacate their office. Only sincere repentence and rejection of his faith can save him from the living nightmares that his former brethren can exact on him.;;;;;;;;;;x
+donatist_redeemed;Prodigal Child;;;;;;;;;;x
+donatist_redeemed_desc;This character was once a Traditor to his Donatist faith, but like the Prodigal Son of old, he has come back into his faith, where he has been welcomed with open arms.;;;;;;;;;;x
```

Comment:
It's the right file. Those are RELIGIOUS modifiers and Traits, so they go in the RELIGION section.